common/spec/dependabot/utils_spec.rb

```diff
@@ -0,0 +1,175 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "dependabot/utils"
+
+RSpec.describe Dependabot::Utils do
+  describe ".wildcard_match?" do
+    subject { described_class.wildcard_match?(wildcard_string, candidate_string) }
+
+    context "without a wildcard" do
+      let(:wildcard_string) { "bus" }
+
+      context "with a matching string" do
+        let(:candidate_string) { wildcard_string }
+        it { is_expected.to eq(true) }
+
+        context "with different capitalisation" do
+          let(:candidate_string) { "Bus" }
+          it { is_expected.to eq(true) }
+        end
+      end
+
+      context "with a superstring" do
+        let(:candidate_string) { wildcard_string + "iness" }
+        it { is_expected.to eq(false) }
+      end
+
+      context "with a substring" do
+        let(:candidate_string) { "bu" }
+        it { is_expected.to eq(false) }
+      end
+
+      context "with a string that ends in the same way" do
+        let(:candidate_string) { "blunderbus" }
+        it { is_expected.to eq(false) }
+      end
+
+      context "with a regex character" do
+        let(:wildcard_string) { "bus." }
+
+        context "with a matching string" do
+          let(:candidate_string) { wildcard_string }
+          it { is_expected.to eq(true) }
+        end
+
+        context "with a superstring" do
+          let(:candidate_string) { wildcard_string + "iness" }
+          it { is_expected.to eq(false) }
+        end
+      end
+    end
+
+    context "with a wildcard" do
+      context "at the start" do
+        let(:wildcard_string) { "*bus" }
+
+        context "with a matching string" do
+          let(:candidate_string) { wildcard_string }
+          it { is_expected.to eq(true) }
+        end
+
+        context "with a matching string (except the wildcard" do
```

Comment:
```suggestion
        context "with a matching string (except the wildcard)" do
```

Nit: balanced parens.